app/main/views/briefs.py

```diff
@@ -162,6 +162,7 @@ def list_briefs():
 
 @main.route('/briefs/<int:brief_id>/status', methods=['PUT'])
 def update_brief_status(brief_id):
+    """Route is deprecated. Use `update_brief_status_by_action` instead."""
```

Comment:
Good idea but probably wont be necessary (looks like this route is only being used once across our apps so not much to keep track of)